crates/noirc_frontend/src/node_interner.rs

```diff
@@ -363,6 +425,14 @@ impl NodeInterner {
         self.structs[&id].clone()
     }
 
+    pub fn get_global_const(&self, stmt_id: &StmtId) -> Option<GlobalConstInfo> {
+        self.global_constants.get(stmt_id).cloned()
+    }
+
+    pub fn get_all_global_consts(&self) -> HashMap<StmtId, GlobalConstInfo> {
```

Comment:
I mainly did it this way because I was trying to follow the functionality of the rest of the `NodeInterner` getter functions. In `statement`, `expression`, and `get_struct` we clone before returning. I can definitely still switch it, but perhaps then it would be worth it to also change the rest in a separate issue.